read.go

```diff
@@ -166,6 +169,20 @@ func readPixelData(r dicomio.Reader, t tag.Tag, vr string, vl uint32, d *Dataset
 		return nil, errors.New("the Dataset context cannot be nil in order to read Native PixelData")
 	}
 
+	if r.SkipPixelData() {
+		r.Skip(int64(vl))
+		return &pixelDataValue{
+			PixelDataInfo: PixelDataInfo{
+				Frames: []frame.Frame{
+					{
+						Encapsulated: false,
+						NativeData:   frame.NativeFrame{},
+					},
+				},
+			},
+		}, nil
+	}
```

Comment:
Something to consider, should we still emit an empty PixelData element? I suppose it may still be useful to know the PixelData element _did_ exist. In this case, it may also be useful to have a marker inside PixelDataInfo that indicates that pixel data was intentionally not parsed. 

Consider adding `PixelDataInfo.DataIntentionallyNotParsed` bool or something.

You may want to handle this behavior directly in [`readPixelData`](https://github.com/suyashkumar/dicom/blob/836cc53a9bb7a481e1acf91e53a9ca4fde1149f4/read.go#L126) instead of in readRawItem and readNativePixelData. Feels a bit odd to have readRawItem concerned with this.

Reply:
Actually knowing the PixelData element did exist is very important in my case. I just don't care about the content.
Thank you both for creating the library and adding this option to optimize performance. In my case 97% of the memory that my application is using stems from the Pixel Data that I don't need.

Reply:
@suyashkumar these changes are already in `readPixelData`?